app/code/Magento/Store/Model/StoreManager.php

```diff
@@ -252,7 +280,12 @@ public function getDefaultStoreView()
     }
 
     /**
-     * {@inheritdoc}
+     * Retrieve application store group object
```

Comment:
Please revert the comment and just remove curly braces `{}`. Please refer to the guide https://devdocs.magento.com/guides/v2.3/coding-standards/docblock-standard-general.html#inheritdoc